src/ipyvizzustory/storylib/story.py

```diff
@@ -229,11 +251,12 @@ def is_pixel(value: Any) -> bool:
             `True` if the value is pixel, `False` otherwise.
         """
 
-        value_is_pixel = False
-        if isinstance(value, str):
-            if value.endswith("px"):
-                value_is_pixel = value[:-2].isnumeric()
-        return value_is_pixel
+        if StorySize._is_int(value) or StorySize._is_float(value):
```

Comment:
Because of `__init__` argument converted ints and floats to strings,  these conditions will always be false.

Reply:
Is it used outside this class? If not, make it private "_is_pixel" then remove this check, because it is a class invariant. If we want to use outside a class, no change required

Reply:
It was public in the previous released version, I don't want to change this in the next minor release.